target/linux/bcm27xx/image/Makefile

```diff
@@ -51,6 +51,10 @@ define Build/boot-2711
 endef
 
 define Build/sdcard-img
+	if [ -d $(TOPDIR)/overlay/boot ]; then \
```

Comment:
considering boot contains an ./overlays directory... this is a really confusing choice for path names under TOP_DIR

```
./overlay/boot/overlays/something.dts
```

suggest files_boot or files-bootpart